data-model/nginx/nginx.conf

```diff
@@ -0,0 +1,17 @@
+worker_processes 4;
+
+events {
```

Comment:
Do we need this or can it be removed?

Reply:
I think you need it with an http block. Without it, the container crashes: 

```
[emerg] 1#1: no "events" section in configuration
nginx: [emerg] no "events" section in configuration
```

Reply:
Ok wasn't sure, thanks for checking